app/src/main/java/com/eventersapp/marketplace/util/AppConstants.kt

```diff
@@ -1,13 +1,16 @@
-package com.eventersapp.consumer.util
+package com.eventersapp.marketplace.util
 
 object AppConstants {
 
-    //<editor-fold desc="">
+    //<editor-fold desc="Bottomsheet Tag">
+
+    const val TAG_SHOW_QR_CODE_BOTTOM_SHEET_FRAGMENT = "show_qr_code_bottom_dialog_fragment"
 
     //</editor-fold>
 
     //<editor-fold desc="Commonly used constants">
 
+    const val S3URL = "<S3-URL>"//Replace with S3 url
     const val BLOG_KEY = "<key>"//Replace with ghost blog key
```

Comment:
Add this in Readme.md, to replace s3url and blog key

Reply:
Done